src/analyzer.ts

```diff
@@ -249,7 +249,6 @@ export class BuildAnalyzer {
       if (this.config.isSource(filePath)) {
         const err = new Error(`Not found: ${filePath}`);
         this.loader.rejectDeferredFile(filePath, err);
-        return;
```

Comment:
This early return is unintuitive. It seems dangerous to not process the remaining errors or close the dependencies steam. I also don't see how the test should show more errors with this addition. Does the modified test fail without this change?

Reply:
I think the change removes the early return.

Reply:
Right, so I shouldn't do reviews on my phone on the subway :)